README.md

```diff
@@ -35,7 +35,17 @@ A very common misconception is that quality takes time and money. Actually, anyt
 ## Maintenance and Contribution
 QMM is an [open-source project](https://github.com/qualiticcommunity/qualiticcommunity.github.io) and its maintainers highly try to avoid any exercise of authority, while keeping it usable, maintainable, and practical.  
 So contributions of any kind to the maturity model - through opening issues, pull requests, or contacting us any other way - are appreciated.  
-Feel free to let us know how your experience using this maturity model was, and what can make it better for your team/organization.
+
+If you like QMM, please share it! And let us know how your experience using this maturity model was, and what can make it better for your team/organization.  
```

Comment:
I suggest this:
If you like QMM, please support us by sharing it.

Reply:
I applied your suggestion but kept the next sentence too. Please review it again. thanks.